mmhuman3d/data/data_converters/vibe.py

```diff
@@ -0,0 +1,202 @@
+            std=[58.395, 57.12, 57.375],
+            to_rgb=True)
+        self.transforms = Compose([
+            dict(type='LoadImageFromFile'),
+            dict(type='MeshAffine', img_res=224),
+            dict(type='Normalize', **self.img_norm_cfg),
+            dict(type='ImageToTensor', keys=['img']),
+        ])
+        self.device = torch.device(
+            'cuda') if torch.cuda.is_available() else torch.device('cpu')
+        self.batch_size = 128
+        self.pretrained_ckpt = pretrained_ckpt
+
+    @staticmethod
+    def init_model(model_config, ckpt_path, device):
+        config = mmcv.Config.fromfile(model_config)
+        # modify config to support single gpu/cpu
+        config.model.backbone.norm_cfg = {'type': 'BN', 'requires_grad': True}
+        model = build_architecture(config.model)
+
+        head_keys = [
+            'fc1.weight', 'fc1.bias', 'fc2.weight', 'fc2.bias',
+            'decpose.weight', 'decpose.bias', 'decshape.weight',
+            'decshape.bias', 'deccam.weight', 'deccam.bias'
+        ]
+
+        def key_transformation(old_key):
+            if 'extractor' in old_key:
+                old_key = old_key.replace('extractor', 'backbone')
+            if old_key in head_keys:
+                old_key = 'head.' + old_key
+            return old_key
+
+        if ckpt_path is not None:
+            state_dict = torch.load(ckpt_path)['state_dict']
+            new_state_dict = OrderedDict()
+
+            for key, value in state_dict.items():
+                new_key = key_transformation(key)
+                new_state_dict[new_key] = value
+
+            model.load_state_dict(new_state_dict, strict=False)
+        return model.to(device)
+
+    def convert_by_mode(self, dataset_path: str, out_path: str,
+                        mode: str) -> dict:
+        """
+        Args:
+            dataset_path (str): Path to directory where spin preprocessed
+            npz files are stored
+            out_path (str): Path to directory to save preprocessed npz file
+            mode (str): Mode in accepted modes
+
+        Returns:
+            dict:
+                A dict containing keys image_path, bbox_xywh, keypoints2d,
+                keypoints2d_mask,stored in HumanData() format. keypoints3d,
+                keypoints3d_mask, smpl are added if available.
+
+        """
+        # use HumanData to store all data
+        human_data = HumanData()
+
+        image_path_, keypoints2d_, bbox_xywh_ = [], [], []
+
+        seq_file = self.mapping_dict[mode]
+        seq_path = os.path.join(dataset_path, seq_file)
+        data = np.load(seq_path)
+
+        image_path_ = data['imgname']
+        num_data = len(image_path_)
+
+        if mode == 'mpi_inf_3dhp':
+            image_path_ = np.array([
+                s.replace('imageFrames/', '').replace('frame_', '')
+                for s in data['imgname']
+            ])
+
+        # center scale to bbox
+        w = h = data['scale'] * 200
```

Comment:
Just to confirm this `200` is to off-set the magic number in the original dataset?

Reply:
Yes, this is to account for data with pixel_std=200 in SPIN preprocessed datasets